R/post_alignment.R

```diff
@@ -238,4 +238,42 @@ setOtherPrecursors <- function(df, refIdx, XICs, analytes, params){
   invisible(NULL)
 }
 
+#' Populate Alignment Feature Mapping Table
+#'
+#' Populate an alignment feature mapping table for star align method,
+#' to map aligned features in experiment runs to the reference run for a given analyte.
+#' Is a data.table generated from of \code{\link{getRefExpFeatureMap}}.
+#' @author Justin Sing, \email{justinc.sing@mail.utoronto.ca}
+#'
+#' ORCID: 0000-0003-0386-0092
+#'
+#' License: (c) Author (2020) + GPL-3
+#' Date: 2022-11-07
+#' @importFrom data.table rbindlist as.data.table set
+#' @inheritParams alignTargetedRuns
+#' @param df (dataframe) a collection of features related to the peptide
+#' @param feature_alignment_mapping (data.table)  contains experiment feature ids
+#' mapped to corresponding reference feature id per analyte. This is an output of \code{\link{getRefExpFeatureMap}}.
+#' @param tAligned (matrix) the first column corresponds to the aligned reference time, the second column is the aligned experiment time. #' @seealso \code{\link{getAlignedTimesFast}}
+#' @param ref (string) name of the reference run. Must be in the rownames of fileInfo.
+#' @param eXp (string) name of the run to be aligned to reference run. Must be in the rownames of fileInfo.
+#' @param analyte_chr (string) name of highest quality precursor/transition_group_id
+populateReferenceExperimentFeatureAlignmentMap <- function(df, feature_alignment_mapping, tAligned, ref, eXp, analyte_chr){
```

Comment:
Need to specify if this is internal function or exported one. 
Also, having an example would help if exported.

Reply:
Okay, this is internal.